src/main/resources/anniv5Resources/localization/eng/coresetpack/Cardstrings.json

```diff
@@ -22,9 +22,7 @@
   },
   "${ModID}:RippedPecs": {
     "NAME": "Ripped Pecs",
-    "DESCRIPTION": "Gain 1 Strength for each Pack you've played a card from this combat. NL Exhaust.",
-    "UPGRADE_DESCRIPTION": "Gain 1 Strength for each Pack you've played a card from this combat.",
-    "EXTENDED_DESCRIPTION": [" NL (Gains {0} Strength.)"]
+    "DESCRIPTION": "Gain 1 Strength. NL ${ModID}:Synergy: Gain !M! Strength."
```

Comment:
Having this be 2 power applications feels a bit jank. Might be nicer to say Gain !M! Strength instead and change the use method accordingly.

Reply:
I'm confused, do you mean that it should not have the 1 strength by default?  or that the use code should total the two, and the text here is correct?